docs/hopsworksai/autoscaling.md

```diff
@@ -0,0 +1,67 @@
+# Autoscaling
+If you run a Hopsworks cluster version 2.2 or above you can enable autoscaling to let hopsworks.ai start and stop workers depending on the demand.
+
+## Enabling and configuring the autoscaling
+Once you have created a cluster you can enable autoscaling by going to the *Details* tab and clicking on *Configure autoscale*
+
+<p align="center">
+  <figure>
+    <a  href="../../assets/images/hopsworksai/configure_autoscale.png">
+      <img style="border: 1px solid #000" src="../../assets/images/hopsworksai/configure_autoscale.png" alt="Configure autoscale">
+    </a>
+    <figcaption>Configure autoscale</figcaption>
+  </figure>
+</p>
+
+Once you have clicked on *Configure autoscale* you will access a form allowing you to configure the autoscaling. This form is in two parts. In the first part, you configure the autoscaling for general-purpose compute nodes. In the second part, you configure the autoscaling for nodes equipped with GPUs. In both part you will have to set up the following:
+
+1. The instance type you want to use. You can decide to not enable the autoscaling for GPU nodes by selecting *No GPU autoscale*.
+2. The size of the instances' disk.
+3. The minimum number of workers. 
+4. The maximum number of workers.
+5. The targeted number of overprovisioned nodes. Setting resource overprovisioning ensures that there are always some free resources in your cluster. This ensures that requests for new resources are fulfilled promptly. You configure the overprovisioning by setting the number of nodes you want to be overprovisioned. For example, if you set a value of *0.5* the system will start a new worker each time there are less than the resource equivalent of half a worker in the cluster. If you set this value to 0 new workers will only be started when a job or notebook request the resources.
+6. The time to wait before removing unused resources. One often starts a new computation shortly after finishing the previous one. To avoid having to wait for workers to stop and start between each computation it is recommended to wait before shutting down workers. Here you set the amount of time in seconds resource need to be unused before they get removed from the system.
```

Comment:
```suggestion
6. The time to wait before removing unused resources. One often starts a new computation shortly after finishing the previous one. To avoid having to wait for workers to stop and start between each computation it is recommended to wait before shutting down workers. Here you set the amount of time in seconds resources need to be unused before they get removed from the system.
```